test/integrations/destinations/facebook_pixel/processor/pageScreenTestData.ts

```diff
@@ -0,0 +1,577 @@
+
+const commonScreenMessage = { ...commonMessage, type: 'screen' };
+
+const commonStatTags = {
+  errorCategory: 'dataValidation',
+  errorType: 'instrumentation',
+  destType: 'FACEBOOK_PIXEL',
+  module: 'destination',
+  implementation: 'native',
+  feature: 'processor',
+};
+
+export const pageScreenTestData: ProcessorTestData[] = [
+  {
+    id: 'facebook_pixel-page-test-1',
+    name: 'facebook_pixel',
+    description:
+      'Page call : Happy flow without standard page switched on and with name and properties',
+    scenario: 'Page',
+    successCriteria: 'Response should contain status code 200 and no error message',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: commonPageMessage,
+            metadata: generateMetadata(1),
+            destination: commonDestination,
+          },
+        ],
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            output: transformResultBuilder({
+              version: '1',
+              type: 'REST',
+              method: 'POST',
+              endpoint: `https://graph.facebook.com/${VERSION}/dummyPixelId/events?access_token=09876`,
+              headers: {},
+              params: {},
+              FORM: {
+                data: [
+                  '{"user_data":{"external_id":"5994471abb01112afcc18159f6cc74b4f511b99806da59b3caf5a9c173cacfc5","client_ip_address":"0.0.0.0","client_user_agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36"},"event_name":"Viewed page ApplicationLoaded","event_time":1697278611,"event_source_url":"jkl","event_id":"5e10d13a-bf9a-44bf-b884-43a9e591ea71","action_source":"website","custom_data":{"path":"/abc","referrer":"xyz","search":"def","title":"ghi","url":"jkl"}}',
+                ],
+              },
+              files: {},
+              userId: '',
+            }),
+            statusCode: 200,
+            metadata: generateMetadata(1),
+          },
+        ],
+      },
+    },
+  },
+  {
+    id: 'facebook_pixel-page-test-2',
+    name: 'facebook_pixel',
+    description: 'Page call : with standard page switched on and no properties and no name',
+    scenario: 'Page',
+    successCriteria:
+      'Response should contain error message and status code should be 400, as we are not sending any other properties other than standard page properties',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: generateSimplifiedPageOrScreenPayload(
+              {
+                event: 'TestEven001',
+                sentAt: '2021-01-25T16:12:02.048Z',
+                userId: 'sajal12',
+                context: {
+                  traits: {},
+                },
```

Comment:
Any reason for passing empty `traits`?

Reply:
no reason, just wanted to keep it short.

Reply:
updated with minimal values